routes/web.php

```diff
@@ -15,12 +15,9 @@
     return view('auth.login');
 });
 
-Route::get('/category', function () {
-    return view('category');
-});
-
 Auth::routes();
 
 Route::get('/home', 'HomeController@index')->name('home');
 Route::get('/users', 'UserController@users');
+Route::get('/user/{id}/profile', 'UserController@show')->name('profile');
 // Route::get('/users', 'UserController@users');
```

Comment:
Remove the `// Route::get('/users', 'UserController@users');`, this is a duplicate, you already have something the same above.